plaidml/bridge/openvino/plugin/CMakeLists.txt

```diff
@@ -1,5 +1,7 @@
 set(TARGET_NAME "plaidmlPlugin")
 
+ie_cpack_set_library_dir()
```

Comment:
What was this added for?

Reply:
After one commit in OV, plugin has to explicitly set LIBRARY PATH, otherwise when cmake will give this error:
```
CMake Error at /home/yangleizou/plaidml/openvino/cmake/developer_package/plugins/plugins.cmake:107 
(install):
  install TARGETS given no LIBRARY DESTINATION for module target
  "plaidmlPlugin".
Call Stack (most recent call first):
  plaidml/bridge/openvino/plugin/CMakeLists.txt:3 (ie_add_plugin)
```
the function definition is here:
https://github.com/openvinotoolkit/openvino/blob/master/cmake/developer_package/packaging.cmake#L14